TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/cluster_curl/setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import setup, find_packages  # ignore: type
+
+setup(
+    name="cluster_curl",
+    version="1.0.0",
+    description="A Python tool to make API calls against congfigured clusters",
+    packages=find_packages(exclude=("tests")),
+    install_requires=["console_link", "Click"],
+    entry_points={
+        "console_scripts": [
+            "ccurl = cli:cli",
```

Comment:
I'd choose another name other than curl as the root name.  With curl, a user might think that they can use --verbose to trace activity, append a binary file,  set cookies, etc.  This is useful by itself.  It doesn't need to try to replace or compete with curl.